src/ISA/ISA.Spreadsheet/CompositeCell.fs

```diff
@@ -0,0 +1,24 @@
+﻿module ISA.Spreadsheet.CompositeCell
+
+open ISA
+open FsSpreadsheet
+
+let fromFsCells (cells : list<FsCell>) : CompositeCell =
+    let cellValues = cells |> List.map (fun c -> c.Value)
+    match cellValues with
+    | [v] -> CompositeCell.createFreeText v
+    | [v1;v2;v3] -> CompositeCell.createTermFromString(v1,v2,v3)
+    | [v1;v2;v3;v4] -> CompositeCell.createUnitizedFromString(v1,v2,v3,v4)
+    | _ -> 
+        failwithf "Dafuq"
+
+let toFsCells isTerm hasUnit (cell : CompositeCell) : list<FsCell> =
+    match cell with
```

Comment:
What is happening here? Why is `isTerm`/`hasUnit` given from outside? As this should be coming only from the CompositeCell?

If this must stay here, pls use the `ToCompositeCellType()` functions first to reduce code duplication:
https://github.com/nfdi4plants/ISADotNet/blob/8d174a3178e115aa16a8f4538e225626d68a497c/src/ISA/ISA/ArcTypes/CompositeCell.fs#L26

So have a `TermToFsCell`, `FreetextToFsCell`, `UnitizedToFsCell` subfunction which you call after using the conversion functions referenced above!